scripts/aws/ebs-warmer.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Ref: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-initialize.html
+
+function log() {
+    echo "[$(date --rfc-3339=seconds)] $1"
+}
+
+log "Warming up RWD EBS volume..."
+find /opt/rwd-data/nhd/Main_Watershed/* -type f | xargs -L1 -t cat >/dev/null
+find /opt/rwd-data/drb/Main_Watershed/* -type f | xargs -L1 -t cat >/dev/null
```

Comment:
I think that `shellcheck` just wants you to add `-print0` to `find` and `-0` to `xargs` so that it can more confidently parse the file names that are output:

```bash
$ find /opt/rwd-data/drb/Main_Watershed/* -type f -print0 | xargs -0 -L1 -t cat >/dev/null
```